lambdatimes/src/components/Content/Content.js

```diff
@@ -17,38 +17,30 @@ export default class Content extends Component {
   }
 
   componentDidMount() {
-    // Once the component has mounted, get the data and reflect that data on the state.
+    this.setState({
+      tabs: tabData,
+      cards: cardData
+    });
   }
 
   changeSelected = tab => {
-    // this function should take in the tab and update the state with the new tab.
+    this.setState({
+      selected: tabData
+    });
   };
 
   filterCards = () => {
-    /* Right now this function only returns the cards on state.
-      We're going to make this function more dynamic
-      by using it to filter out our cards for when a tab is selcted
-      
-      Notice that we're passing this function to our <Cards /> component below.
-      This function returns an array of cards, so we can just pass it down as such.
-
-      Your algorithim for the logic here is as follows: 
-        - if the selected tab is 'all' it should return all 
-          of the items from cardData. 
-        - else, it should only return those cards whose 'tab' matched this.state.selected.
-    */
-    return this.state.cards;
+    if(this.state.selected === 'all'){
+      return this.state.cards;
+    } else{
+    return this.state.cards.filter(card => {return card.tab === this.state.selected});
+    }
   };
 
   render() {
     return (
       <div className="content-container">
-        {/* 
-          Add 2 props to the Tabs component, 
-          `selectedTab` that includes the currently selected tab
-          and `selectTabHandler` that includes the function to change the selected tab
-        */}
-        <Tabs tabs={this.state.tabs} />
+        <Tabs tabs={this.state.tabs}  selectedTab={this.state.selected} selectHandler ={this.changeHandler} />
```

Comment:
you need to change the name of the selectHandler prop you are passing, it is not called changeHandler, that is why your function is not working.